sys/dev/mfi/mfi_ioctl.h

```diff
@@ -57,13 +57,13 @@ union mfi_statrequest {
 #define MAX_SPACE_FOR_SENSE_PTR		32
 union mfi_sense_ptr {
 	uint8_t		sense_ptr_data[MAX_SPACE_FOR_SENSE_PTR];
-	void 		*user_space;
+	void * __kerncap user_space;
```

Comment:
nit. Keep indentation consistent with rest of file
```suggestion
	void			* __kerncap user_space;
```